.github/workflows/unitctl.yml

```diff
@@ -15,6 +15,50 @@ permissions:
   contents: write
 
 jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: tools/unitctl
+    env:
+      MAKE: make
+      CARGO: cargo
+      VERSION:
+      SHORT_VERSION:
```

Comment:
If we aren't doing cross we shouldn't need these, correct? The VERSION stuff is for releasing either way.

Reply:
we are using cross!